contrib/alertmanager-arm64/v0.15.0/Makefile

```diff
@@ -1,2 +0,0 @@
-build:
```

Comment:
Curious why the makefile was deleted?

Reply:
Renamed the folder to match the others, because it had that 'v' prefix before the tag and all the other folders didn't have it.